app/ui/browser/reducers/ui-state.js

```diff
@@ -55,6 +55,17 @@ export default function uiState(state = initialState, action) {
     case types.SET_URL_INPUT_AUTOCOMPLETE_INDEX:
       return state.set('focusedResultIndex', action.payload.index);
 
+    case types.TOGGLE_PAGE_SUMMARIES: {
```

Comment:
We've decided in the past to not use TOGGLE_* as an action, instead preferring SET_* so that the action log / frontend is more explicit about what it's requesting.  See something like SET_URL_INPUT_VISIBLE as an example